accounting/off_chain.go

```diff
@@ -50,6 +59,9 @@ type OffChainConfig struct {
 	// payments that are made to our own node.
 	OwnPubKey route.Vertex
 
+	// DecodePayReq decodes a payment request.
+	DecodePayReq func(payReq string) (*lndclient.PaymentRequest, error)
```

Comment:
nit: add a function type for this to avoid having the full signature in 3 places?